Source/init.cpp

```diff
@@ -223,32 +223,32 @@ void init_await_mom_parent_exit()
 
 void init_archives()
 {
-	HANDLE a1; // [esp+8h] [ebp-8h]
+	HANDLE fh;
 #ifdef COPYPROT
-	int v1; // [esp+Ch] [ebp-4h]
+	int result;
 #endif
-
-	fileinfo.size = 20;
+	memset (&fileinfo, 0, sizeof (fileinfo));
+	fileinfo.size = sizeof (fileinfo);
 	fileinfo.versionstring = gszVersionNumber;
 	fileinfo.executablefile = diablo_exe_path;
 	fileinfo.originalarchivefile = diabdat_mpq_path;
 	fileinfo.patcharchivefile = patch_rt_mpq_path;
 	init_get_file_info();
 #ifdef COPYPROT
-	while (1) {
+	while (TRUE) {

```

Comment:
I think we ended up deciding that (1) was a more likely style for the time.

Reply:
noobs didn't know about for(;;) :(

Reply:
```suggestion
	while (1) {
```